gradle.properties

```diff
@@ -1,7 +1,7 @@
 projectVersion=1.0.0.BUILD-SNAPSHOT
 micronautDocsVersion=1.0.22
 micronautBuildVersion=1.0.0
-micronautVersion=1.3.0.BUILD-SNAPSHOT
+micronautVersion=2.0.0.M3
```

Comment:
Any reason to not cut a release before increasing this to 2.x? Should this be bumped to `2.0.0`?

Reply:
M3 was the latest version when this PR was created. So yeah, `2.0.0` should be.

However, I don't think this module is ready for a release, nor it means it doesn't work with Micronaut 2.0.

Reply:
The absence of issues promoted the question about cutting a release. Is this library not being used in a meaningful way yet? Are there any known outstanding tasks?

Reply:
This library has never been released, hence the absence of issues.

Regarding outstanding tasks, @jameskleeh would know better.